pkg/util/util.go

```diff
@@ -108,15 +108,15 @@ func IsTestMode() (isTesting bool) {
 
 func GetClusterPublicHostname(isOpenShift4 bool) (hostname string, err error) {
```

Comment:
If you're refactoring `GetClusterPublicHostnameForOpenshiftV4` to `getClusterPublicHostnameForOpenshiftV4` should you also lowercase the `get` in this method name too?

Reply:
in fact upper-case in Go means *public*, which is why I only kept this method with a upper-case.

Reply:
ugh, that's about as non-obvious to a new user as syntactic whitespace in yaml. Not a fan. :(

Reply:
not a fan either, TBH